### PR TITLE
Update aws.py

### PR DESCRIPTION
**Pull request by xiaowei-zillow**
add function for users to send message to aws sqs

### CODE REVIEW

File: zdatasets/utils/aws.py [hunk 1]
```diff
@@ -63,6 +60,46 @@ def get_aws_client(role_arn: str, service: str):
     return session.client(service)
 
 
+def send_sqs_message(
+    queue_url: str,
+    message_body: str,
+    *,  # keyword-only parameters
```
Comment (talebzeghmi) [inline zdatasets/utils/aws.py:66]: is keyword-only parameters necessary given the following parameters are already keyword only?
Maybe I don't understand it's use.

Comment (xiaowei-zillow) [reply zdatasets/utils/aws.py:66]: If without *, it is confusing what does `val3` means in this call, send_sqs_message(val1, val2, val3). '*' is telling that all parameters after it must be keyword-only.

Comment (talebzeghmi) [reply zdatasets/utils/aws.py:66]: Must they not be named anyways because role_arn and profile_name are named?
Or are you introducing `*` as positional parameter to force any future parameters after it to be keyword only? :)

File: zdatasets/utils/aws.py [hunk 2]
```diff
@@ -63,6 +60,46 @@ def get_aws_client(role_arn: str, service: str):
     return session.client(service)
 
 
+def send_sqs_message(
+    queue_url: str,
```
Comment (aaron-arellano) [inline zdatasets/utils/aws.py:64]: do you have another PR that shows where the config will be input?

File: zdatasets/utils/aws.py [hunk 3]
```diff
@@ -63,6 +60,46 @@ def get_aws_client(role_arn: str, service: str):
     return session.client(service)
 
 
+def send_sqs_message(
```
[the remark this answers is not in the document]
Comment (xiaowei-zillow) [reply zdatasets/utils/aws.py:63]: we will update exit_handler, so send_sqs_message method will be called if sqs url is defined in kubeflow profile.

File: zdatasets/utils/aws.py (hunk 2 above)
Comment (xiaowei-zillow) [reply zdatasets/utils/aws.py:64]: yes, will have separate PRs in repo profile, webhook and kfp etc.

**Review by talebzeghmi — APPROVED** [review]
